jovian/jovian_nb_ext/main.js

```diff
@@ -590,16 +594,16 @@ define([
               $("#nb_filename_box").val(
                 jvn_params.nb_filename.replace(".ipynb", "")
               );
-              jvn_params.secret.toLocaleUpperCase().substr(0, 1) == "F"
+              jvn_params.secret == "False"
```

Comment:
From where "False" string gets picked up? Would the user need to enter it?

Reply:
Its taken from the user via html form, he has made radio button for all the bool parameters.